pos_menu/static/src/js/pos.js

```diff
@@ -0,0 +1,26 @@
+odoo.define('pos_menu', function(require) {
+    var models = require('point_of_sale.models');
+
+    models.load_models([{
+        model: 'pos.tag',
+        field: [],
+        domain: null,
+        loaded: function(self, tags) {
+            self.tags = [];
+            // save tags of current POS
+            tags.forEach(function(tag){
+                if ($.inArray(self.config.id, tag.pos_ids) !== -1) {
```

Comment:
Why don't you filter tags out in ``domain``?

Reply:
@yelizariev it's not a required code. I save the tags of the current POS for future use. What do you mean "filter tags out in domain"? How can I do it?

Reply:
@yelizariev now I understand what you mean. I need use '''domain''', no problem)